testing_and_setup/compass/ocean/drying_slope/init_idealized_transect.xml

```diff
@@ -0,0 +1,18 @@
+<template>
+	<namelist>
+		<option name="config_idealized_transect_Lcoast">0.3</option>
```

Comment:
May need to set `config_use_idealized_transect` here?

Reply:
Good point, thanks!